x/slashrefund/keeper/deposit_test.go

```diff
@@ -66,14 +39,87 @@ func TestDepositRemove(t *testing.T) {
 
 func TestDepositGetAll(t *testing.T) {
```

Comment:
Should be renamed in TestGetAllDeposit since the tested function is GetAllDeposit().

Reply:
right